mbus-api/protobuf/v1/event.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+import "google/protobuf/wrappers.proto";
+
+package v1.event;
+
+// Event Message
+message EventMessage {
+    Category category = 1;
+    Action action = 2;
+    string target = 3;
+    EventMeta metadata = 4;
+}
+
+// Event Category
+enum Category {
+  Unknown = 0;
+  Pool = 1;
+  Volume = 2;
+}
+
+// Event Action
+enum Action {
+  UnknownEvent = 0;
+  CreateEvent = 1;
+  DeleteEvent = 2;
+}
+
+// Event meta data
+message EventMeta {
+    // Something that uniquely identifies events. UUIDv4. GUID.
+    string id = 1;
+    EventSource source = 2;
+    string event_timestamp = 3;
+    string version = 4;
```

Comment:
should this be an enum?

Reply:
What could be the enum variants? UnknownVersion, v1?